packages/framework/src/Form/Admin/Store/StoreFormType.php

```diff
@@ -182,6 +197,7 @@ public function configureOptions(OptionsResolver $resolver): void
             ->setDefaults([
                 'data_class' => StoreData::class,
                 'attr' => ['novalidate' => 'novalidate'],
+                'constraints' => new Constraints\Callback($this->validateOpeningHours(...)),
```

Comment:
this callback validator breaks any js validation, or?

Reply:
that is a good catch! Even more, the JS was already broken by another missing callback validation, fixed in 3be2397433